src/logDownloaderService.py

```diff
@@ -0,0 +1,79 @@
+import os
+import time
+import hashlib
+import subprocess
+import psutil
+from git import Repo
+import shutil
+from loguru import logger
+
+LOGS_DIRECTORY = os.getenv('LOGS_DIRECTORY', '/path')
+ALL_LOGS_FLAG = os.getenv('ALL_LOGS', 'false').lower() == 'true'
```

Comment:
```suggestion
SHOULD_FETCH_NON_MATCH_LOGS = os.getenv('ALL_LOGS', 'false').lower() == 'true'
```
It doesn't matter if that's a flag, that's an implementation detail.